Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -674,7 +674,7 @@ private class SettingsTableSectionHeaderView: UITableViewHeaderFooterView {
         layer.addSublayer(bottomBorder)
     }
 
-    private override func layoutSubviews() {
+   internal override func layoutSubviews() {
```

Comment:
You can delete internal and leave off the modifier since by default this will be internal